docs/the-guide/chapter-7-track-model-evolutions-in-the-cicd-pipeline-with-cml/index.md

```diff
@@ -432,7 +441,7 @@ merge requests (MRs) - to integrate the work done into the `main` branch.
 
 	      echo "## Roc curve" >> report.md
 	      echo >> report.md
-	      dvc plots diff \
+	      poetry run dvc plots diff \
```

Comment:
Why do we need to use `poetry run` all of a sudden?

Reply:
We do not need it.